repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -1498,18 +1498,13 @@ private EntityMutationResponse createOrUpdate(EntityStream entityStream, boolean
                         }
 
                         AtlasEntity diffEntity = reqContext.getDifferentialEntity(entity.getGuid());
-
-                        if (diffEntity != null &&
-                                MapUtils.isNotEmpty(diffEntity.getRelationshipAttributes()) &&
-                                diffEntity.getRelationshipAttributes().containsKey("meanings") &&
-                                diffEntity.getRelationshipAttributes().size() == 1 &&
-                                MapUtils.isEmpty(diffEntity.getAttributes()) &&
-                                MapUtils.isEmpty(diffEntity.getCustomAttributes()) &&
-                                MapUtils.isEmpty(diffEntity.getBusinessAttributes()) &&
-                                CollectionUtils.isEmpty(diffEntity.getClassifications()) &&
-                                CollectionUtils.isEmpty(diffEntity.getLabels())) {
+                        boolean skipAuthBaseConditions = diffEntity != null && MapUtils.isEmpty(diffEntity.getCustomAttributes()) && MapUtils.isEmpty(diffEntity.getBusinessAttributes()) && CollectionUtils.isEmpty(diffEntity.getClassifications()) && CollectionUtils.isEmpty(diffEntity.getLabels());
```

Comment:
If diffEntity is null just make it false and continue, applying same check thrice

Reply:
Will make the logic confusing, keeping it as it is for now